### PR TITLE
Fix check_is_R7 error message for non-R7 objects.

### PR DESCRIPTION
**Pull request by jonthegeek**
The sprintf had an extra argument (leftover from copy/paste of the other error message).

### CODE REVIEW

**Comment by hadley** [comment]
Thanks! Would you mind adding a snapshot test?

**Comment by jonthegeek** [comment]
> Thanks! Would you mind adding a snapshot test?

Done!

File: tests/testthat/test-inherits.R [hunk 1]
```diff
@@ -17,4 +17,5 @@ test_that("throws informative error", {
     foo2 <- new_class("foo2")
     check_is_R7(foo1(), foo2)
   })
+  expect_snapshot(error = TRUE, check_is_R7("a"))
```
Comment (hadley) [inline tests/testthat/test-inherits.R:20]: For an single expression, I’d put `error` at the end. 

And all the snapshots should be checked in. What difference are you seeing?

**Comment by hadley** [comment]
Thanks!

**Pull request merged**